src/sagemaker/image_uris.py

```diff
@@ -188,6 +195,12 @@ def retrieve(
             )
         _validate_arg(full_base_framework_version, list(version_config.keys()), "base framework")
         version_config = version_config.get(full_base_framework_version)
+    elif framework in RL_FRAMEWORKS:
+        deprecation_warn(
+            "SageMaker-hosted RL images no longer accept new pull requests and",
+            "April 2024",
+            " Please pass in `image_uri` to use RLEstimator"
```

Comment:
I think the tech writer do not like using `Please do x` in customer messaging, could you ping @judyheflin for review before merging?